### DISCUSSION TITLE
Spine 4.2 errors with latest AXMOL on Mac for iOS

### DISCUSSION THREAD

**Discussion by PCGEngineer** [axmolengine/axmol#2244] [General]
Getting errors when I try to compile the latest spine extension (4.2) with the latest axmol (2.2.1) in MacOS. Do I have the right latest version of both? Latest Spine extension looks like June 8 2024, v4.2
<img width="1188" alt="Screenshot 2024-11-24 at 1 06 39 PM" src="https://github.com/user-attachments/assets/62a761c3-6a63-46e2-a7f2-faf43e560fb9">

I got these errors too at first, but I fixed by just commenting them out as they are depreciated anyway:
<img width="986" alt="Screenshot 2024-11-24 at 12 41 29 PM" src="https://github.com/user-attachments/assets/ca528ff6-9c4a-4876-b05c-609b8f491a81">

**Comment by rh101** [comment]
The spine42 branch should now work with Axmol 2.1+:

https://github.com/axmolengine/spine-axmol/tree/spine42

**Reply by PCGEngineer** [reply]
It works great now, thank you so much! Got a beta out to testlfight as a result :)
